### DISCUSSION TITLE
Where to ask SyGuS questions? Is SyGuS outdated for PBE?

### DISCUSSION THREAD

**Discussion by pauljurczak** [cvc5/cvc5#9801] [Q&A] [answered]
What is the "best" place to ask SyGuS questions? Is there another format more relevant to PBE (programming by example) these days?

**Comment by ajreynol** [comment, marked as answer]
I think there may be a mailing list (synthlib@cis.upenn.edu) for general SyGuS questions but it is not frequently used.  You can continue asking questions about SyGuS here.

Some solvers probably accept their own native formats, although I don't know of any other format that is more widely used for PBE.
